hasjob/views/index.py

```diff
@@ -419,6 +419,11 @@ def index(basequery=None, md5sum=None, tag=None, domain=None, location=None, tit
     if data['domain'] and data['domain'] not in db.session:
         data['domain'] = db.session.merge(data['domain'])
     data['show_viewcounts'] = show_viewcounts
+    max_counts = get_max_counts(g.impressions.keys())
+    data['max_impressions'] = int(max_counts['max_impressions'])
+    data['max_views'] = int(max_counts['max_views'])
+    data['max_opens'] = int(max_counts['max_opens'])
+    data['max_applied'] = int(max_counts['max_applied'])
```

Comment:
There's only one max number, to represent the 100% position on the gradient scale.